### DISCUSSION TITLE
alist 使用rclone在windows上挂载，你有好的参数吗

### DISCUSSION THREAD

**Discussion by lhuanyun** [AlistGo/alist#7492] [General]
这是我当前使用的参数
`
& 'C:\Program Files\alistrclone\rclone' mount alist_lan:/ W: --vfs-cache-mode writes --cache-dir c:\alistCache --dir-cache-time 24h --network-mode --no-check-certificate --default-permissions --vfs-read-chunk-size-limit 1G --vfs-read-chunk-size 64M --buffer-size 256M
`

**Comment by lhuanyun** [comment]
感觉链接速度一般般，似乎有一点点卡顿。

**Comment by sangchengfang** [comment]
直接用netmount，github有
